include/swift/AST/ASTContext.h

```diff
@@ -491,6 +491,12 @@ class ASTContext final {
   /// Retrieve the declaration of Swift.Optional<T>.None.
   EnumElementDecl *getOptionalNoneDecl() const;
 
+  /// Retrieve the declaration of _Concurrency.DistributedActorPersona.Local.
```

Comment:
in the WIP PR `DistributedActorPersona` and `DistributedActorPersonality` and `DistributedActorLocalStorage` are sometimes used interchangably, they mean the same thing. Just trying to find a good name still..